pkg/summary/summary.go

```diff
@@ -0,0 +1,83 @@
+package summary
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/go-github/v33/github"
+	"github.com/google/pullsheet/pkg/client"
+	"github.com/google/pullsheet/pkg/repo"
+	"github.com/sirupsen/logrus"
+	"time"
+)
+
+func GeneratePullData(ctx context.Context, c *client.Client, repos []string, users []string, since time.Time, until time.Time) ([]*repo.PRSummary, error) {
```

Comment:
This variable name sucks -- I know because I wrote it. `Data` in particular conveys no additional contextual information. Go functions don't need verb prefixes either.

Just name this `Pulls`.